python/nano/src/bigdl/nano/pytorch/inference/optimizer.py

```diff
@@ -1159,10 +1159,10 @@ def load(path, model: Optional[nn.Module] = None, input_sample=None,
         Load a model from local.
 
         :param path: Path to model to be loaded. Path should be a directory.
-        :param model: Required FP32 model to load pytorch model, it is needed if you accelerated
-               the model with accelerator=None by InferenceOptimizer.trace/
-               InferenceOptimizer.quantize. model should be set to None if you choose
-               accelerator="onnxruntime"/"openvino"/"jit".
+        :param model: Required FP32 model to load pytorch model, it is needed if:
+               1. you accelerated the model with accelerator=None by
+               InferenceOptimizer.trace/InferenceOptimizer.quantize.
```

Comment:
How can user know whether to pass this parameter if they get optimized model by `.optimize()` ?

Reply:
Add more comment for this, thanks!